arch/arm/soc/nordic_nrf5/nrf52/Kconfig.soc

```diff
@@ -26,3 +26,11 @@ config SOC_NRF52840_QIAA
 	select SOC_NRF52840
 
 endchoice
+
+config NRF52_ARM_MPU_ENABLE
```

Comment:
In my opinion this parameter should be considered vendor specific and can be called in any way as far as it makes sense :smile: and respects the dependencies. Said that my preference would be NRF52X_ARM_MPU to maintain the same order used for STM32 and Beetle.